### ISSUE TITLE
Transfer learning tutorial bakes number of classes

### ISSUE THREAD

**Issue by resistor** [pytorch/tutorials#395]
From https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html

> model_ft.fc = nn.Linear(num_ftrs, 2)

This bakes the number of classes to two.  It works fine for the example, but causes an error if the user tries to re-apply it to a new dataset, which I would expect to be a common next step for this tutorial.

I believe the better solution would be:

> model_ft.fc = nn.Linear(num_ftrs, len(class_names))

**Comment by githubssd** [comment]
The reason of error can be the different no. of label categories.

**Comment by jlin27** [comment]
Thanks for the suggestion. Added a comment to clarify that.
